src/components/visualizations/chart/legend/StaticLegend.tsx

```diff
@@ -1,11 +1,12 @@
-// (C) 2007-2019 GoodData Corporation
+// (C) 2007-2020 GoodData Corporation
```

Comment:
Should this file be refactored to ` src/components/visualizations/legend/StaticLegend.tsx`

Reply:
No we shouldn’t.
`chart/legend/` is still a valid location in this case